src/board.cpp

```
@@ -0,0 +1,113 @@
+            grid[discrete_cell->get_x()][discrete_cell->get_y()] = discrete_cell;
+            current_population++;
+        }
+    }
+}
+
+
+vector<vector<DiscreteAutomaton*>> Board::get_grid() {
+    return grid;
+}
+
+void Board::set_board(vector<vector<DiscreteAutomaton*>> grid) {
```

cppcheck

[performance] Function parameter 'grid' should be passed by const reference.

(passedByValue)